modules/QnA/resources/qnaModule.js

```diff
@@ -187,7 +187,8 @@
                 var count = 0;
                 ko.utils.arrayForEach(_this.myObservableArray(), function (thread) {
                     for (var i = 0; i < thread().entries().length; i++) {
-                        if (thread().entries()[i]().getType() !== 'Question' && !thread().entries()[i]().isRead()) {
+                        var entry = thread().entries()[i]();
+                        if (entry.getType() !== 'Question' && !entry.isRead() && !(entry.getType() === 'Announcement' && entry.cuePoint().metadata.State === 'Deleted')) {
                             count++;
                         }
                     }
```

Comment:
we can write: 
if (entry.getType() === 'Answer' && !entry.isRead() || (entry.getType() === 'Announcement' && entry.cuePoint().metadata.State !== 'Deleted')) {
                            count++;
                        }

but I not sure if we want to this and enforce other types like that (maybe we want more types to be included)

Reply:
Your choice, but I found it to be very complex and hard to understand.
It's better now but I don't make much sense of it when reading it